docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -70,115 +72,167 @@ ifdef::qs[]
+
+. Select the namespace to host your Connectors instance and then click *Next*.
 
-. Select the OpenShift Dedicated cluster to host the connector instance. Select the OSD cluster by clicking the box representing the cluster, and click *Next* to complete the step.
+. Configure the core configuration for your connector:
+.. Provide a unique name for the connector. 
+.. Type the *Client ID* and *Client Secret* of the service account that you created for Connectors and then click *Next*.
 
-. In this step you configure the common configuration for your connector. Provide a unique name for the connector. Enter the Client ID and Client Secret of the service account you created in the previous task of this quick start in the *Client ID* and *Client secret* text boxes. If you did set up the {product-kafka} ACL as explained in the previous task, the service account has sufficient rights to produce messages to the topic associated with the connector. Click *Next* to complete the step.
+. Provide connector-specific configuration:
+.. *Data shape Format*: Accept the default, `application/octet-stream`.
+.. *Topic Names*: Type the name of the topic that you created for Connectors (for the quick start example, type *test-topic*).
+.. *Content Type*: Accept the default, `text/plain`.
+.. *Message*: Type the content of the message that you want to send to the topic, for the quick start example, type `Hello World!`.
+.. *Period*: Specify the interval (in milliseconds) at which you want the Connectors instance to sends messages to the Kafka topic. For the quick start example, specify `10000`, to send a message every 10 seconds.
 
-. In this step you provide the connector specific configuration.
-.. Leave the *Data shape Format* to `application/octet-stream`
-.. *Topic Names*: Enter the name of the topic you created in the previous task of this quick start.
-.. *Content Type*: Leave to `text/plain`.
-.. *Message*: Enter the content of the message that you want to send to the topic, for example `Hello World!`.
-.. *Period*: The interval in milliseconds at which events will be sent. Set this to `10000`, which will produce an event every 10 seconds.
+. Optionally, configure the error handling policy for your Connectors instance. For the quick start, select *log* (the Connectors instance sends errors to its log).  +
++
+Other options are *stop* (the Connectors instance shuts down in case of errors), or *dead letter queue* (the Connectors instance sends messages that it cannot handle to a dead letter topic that you define for the Connectors Kafka instance). 
 
-. Finally configure the error handler for the connector. You can choose between *stop* (the connector shuts down in case of errors), *log* (the error is logged) or *dead letter queue* (the events that cannot be handled by the connector are sent to a dead letter topic of your Kafka instance).
-Select `log`.
+.. Click *Next*.
```

Comment:
Should this and the following step be sub-steps of the preceding optional step, or are either/both actually regular steps?